addons/rook/template/base/install.sh

```diff
@@ -86,6 +86,11 @@ function rook() {
     rook_operator_deploy
     rook_set_ceph_pool_replicas
     rook_ready_spinner # creating the cluster before the operator is ready fails
+
+    if [ "$ROOK_MINIMUM_NODE_COUNT" -gt "1" ]; then
+        return 0 # do not create a ceph cluster if it should instead be managed by ekco
```

Comment:
below this is looks like there may be code that ekco will not manage. for example `rook_maybe_migrate_from_longhorn`

it looks like you took into account upgrades but im not entirely convinced upgrades will work in some scenarios.

Reply:
agreed this really doesn't handle migrating from longhorn right now